deploy/chart/devfile-registry/README.md

```diff
@@ -21,17 +21,84 @@ E.g. if your cluster's ingress domain is 192.168.1.0.nip.io, you would run:
 helm install devfile-registry deploy/chart/devfile-registry --set global.ingress.domain=192.168.1.0.nip.io
 ```
 
+### Kubernetes Installation Examples
+```bash
+$ helm install devfile-registry ./deploy/chart/devfile-registry \ 
+    --set global.ingress.domain=<ingress-domain> \
+	--set devfileIndex.image=<index-image> \
+	--set devfileIndex.tag=<index-image-tag>
+```
+
+Where `<ingress-domain>` is the ingress domain for your cluster, `<index-image>` is the devfile index image you want to deploy, and `<index-image-tag>` is the corresponding image tag for the devfile index image.
+
+For example, if you're installing your own custom devfile registry image for dev/test purposes on Minikube, you might run:
+
+```bash
+$ helm install devfile-registry ./deploy/chart/devfile-registry \
+    --set global.ingress.domain="$(minikube ip).nip.io" \
+	--set devfileIndex.image=quay.io/someuser/devfile-index \
+	--set devfileIndex.tag=latest
+```
+
+You can deploy a devfile registry with a custom registry viewer image (uses `quay.io/devfile/registry-viewer:next` by default) by running the following:
+
+```bash
+$ helm install devfile-registry ./deploy/chart/devfile-registry \
+    --set global.ingress.domain="$(minikube ip).nip.io" \
+	--set devfileIndex.image=quay.io/someuser/devfile-index \
+	--set devfileIndex.tag=latest \
+	--set registryViewer.image=quay.io/someuser/registry-viewer \
+	--set registryViewer.tag=latest
+```
+
+You can deploy a *headless* devfile registry (i.e. without the registry viewer) by specifying `--set global.headless=true` which will look like:
+
+```bash
+$ helm install devfile-registry ./deploy/chart/devfile-registry \
+    --set global.ingress.domain="$(minikube ip).nip.io" \
+	--set global.headless=true \
+	--set devfileIndex.image=quay.io/someuser/devfile-index \
+	--set devfileIndex.tag=latest
+```
+
 ## Installing the Devfile Registry on OpenShift
 
-If you're installing on OpenShift, you need to set `global.isOpenShift` to true, for example:
+If you're installing on OpenShift, you will first need to set `global.isOpenShift` to true, for example:
 ```
 helm install devfile-registry deploy/chart/devfile-registry --set global.isOpenShift=true
 ```
 
-or, if you want to install a specific devfile index image, you can run:
+There are 3 ways that you can install the registry on OpenShift:
+#### 1: Via Installation Script With OpenShift Generated Route Hostname and Domain
```

Comment:
We don't necessarily need to cover this now, but something to keep in mind If we eventually onboard the chart to something like ArtifactHub, we would need to provide instructions for installing the chart _without_ the script. (So basically installing the chart, then covering how to get the route, and run `helm upgrade`).

Given that we don't run on artifacthub yet, I don't think we need to worry about that _just_ yet, but good to be aware of it.

Reply:
I think opening an issue for this to make sure it doesn't get lost would be good, wdyt?